lib/spack/docs/basic_usage_tutorial.rst

```diff
@@ -0,0 +1,1243 @@
+  ==> 10 installed packages.
+  -- linux-redhat6-x86_64 / gcc@4.4.7 -----------------------------
+      hdf5@1.10.0-patch1  /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/hdf5-1.10.0-patch1-twppaioxqn6lti4grgopnmhwcq3h2rpw
+      hdf5@1.10.0-patch1  /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/hdf5-1.10.0-patch1-j36yfw25i6gdd3q4vwlupgkpwic4ua6m
+      hdf5@1.10.0-patch1  /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/hdf5-1.10.0-patch1-ezvtnox35albuaxqryuondweyjgeo6es
+      libdwarf@20160507   /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/libdwarf-20160507-dtg3tgnp7htccoly26gduqlrgvnwcp5t
+      libdwarf@20160507   /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/libdwarf-20160507-yfx6p3g3rkmqvcqbmtb34o6pln7pqvcz
+      libelf@0.8.12       /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/libelf-0.8.12-ipggckv6i7h44iryzfa4dwdela32a7fy
+      libelf@0.8.12       /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/libelf-0.8.12-vrv2ttbd34xlfoxy4jwt6qsjrcbalmmw
+      libelf@0.8.13       /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/libelf-0.8.13-csrt4qxfkhjgn5xg3zjpkir7xdnszl2a
+      trilinos@12.8.1     /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/gcc-4.4.7/trilinos-12.8.1-uvd6dfd7x4uyvck4awo3r3frudihn4ar
+
+  -- linux-redhat6-x86_64 / intel@16.0.3 --------------------------
+      libelf@0.8.13  /g/g0/becker33/SC16-spack/spack/opt/spack/linux-redhat6-x86_64/intel-16.0.3/libelf-0.8.13-7wgp32xksatkvw2tbssmehw2t5tnxndj
+
+
+---------------------
+Customizing Compilers
+---------------------
+
+
+Spack manages a list of available compilers on the system, detected
+automatically from from the user's ``PATH`` variable. The ``spack
+compilers`` command is an alias for the command ``spack compiler list``.
+
+.. code-block:: console
+
+  $ spack compilers
+  ==> Available compilers
+  -- gcc ----------------------------------------------------------
+  gcc@4.4.7
+
+  -- intel --------------------------------------------------------
+  intel@16.0.3
+  intel@16.0.2
+  intel@16.0.1
+  intel@16.0.0
+  intel@15.0.4
+  intel@15.0.3
+  intel@15.0.1
+  intel@15.0.0
+  intel@14.0.4
+  intel@14.0.3
+  intel@14.0.2
+  intel@14.0.1
+  intel@14.0.0
+  intel@13.1.1
+  intel@13.1.0
+  intel@13.0.1
+  intel@13.0.0
+  intel@12.1.5
+  intel@12.1.3
+  intel@12.1.2
+  intel@12.1.0
+  intel@12.0.4
+  intel@11.1
+  intel@10.1
+  intel@10.0
+  intel@9.1
+
+  -- pgi ----------------------------------------------------------
+  pgi@16.5-0
+  pgi@16.3-0
+  pgi@16.1-0
+  pgi@15.10-0
+  pgi@15.7-0
+  pgi@15.5-0
+  pgi@15.1-0
+  pgi@14.10-0
+  pgi@14.7-0
+  pgi@14.3-0
+  pgi@13.10-0
+  pgi@13.6-0
+  pgi@13.2-0
+  pgi@13.1-1
+  pgi@12.8-0
+  pgi@12.1-0
+  pgi@11.10-0
+  pgi@11.1-0
+  pgi@10.9-0
+  pgi@10.2-0
+  pgi@9.0-4
+  pgi@8.0-1
+  pgi@7.1-3
+  pgi@7.0-6
```

Comment:
Can these compilers be listed in multiple columns? Or even have fewer versions? It's kind of a lot to scroll through.